packages/trip-form/src/TripOptions/types.d.ts

```diff
@@ -0,0 +1,38 @@
+export interface ModeOption {
+  company?: string;
+  image?: string;
+  hidden?: boolean;
+  label: string;
+  mode: string;
+}
+
+export interface Category {
+  description?: string;
+  image?: string;
+  label: string;
+  mode?: string;
+  options: ModeOption[];
+  type: string;
+}
+
+export interface Modes {
+  categories: Category[];
+  transitModes: ModeOption[];
+}
+
+export interface Company {
+  id: string;
+  label: string;
+  modes: string;
+}
+
+export interface QueryParams {
+  mode?: string;
+  company?: string;
+}
+
+export interface QueryProps {
+  onQueryParamChange(paramsToUpdate: QueryParams): void;
+  queryParams: QueryParams;
+  supportedModes: Modes;
+}
```

Comment:
Add JSDocs so they appear in storybook in the docs pane for the component's story.

Reply:
Added in 6d0d62dd6cd2cba86e6fab24c6c8d7222ee0e239, but @landonreed could you possibly expand since I'm not entirely sure what some of them do